### DISCUSSION TITLE
Copilot doesn't seem to do anything for me other than autocomplete comments

### DISCUSSION THREAD

**Discussion by AdamDSherman** [community/community#43163] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I signed up to trial Copilot and have been (trying) to use it for a month and with web projects (mostly JS).

But it doesn't seem to do anything other than autocomplete comments.

How do I use it?

**Comment by jfullstackdev** [comment]
maybe this will help 

https://www.youtube.com/watch?v=DeO7xLXORpY
